components/resources/library/src/macosMain/kotlin/org/jetbrains/compose/resources/ResourceReader.macos.kt

```diff
@@ -39,12 +39,15 @@ internal actual fun getPlatformResourceReader(): ResourceReader = object : Resou
     private fun getPathOnDisk(path: String): String {
         val fm = NSFileManager.defaultManager()
         val currentDirectoryPath = fm.currentDirectoryPath
+        val pathFix = path.removePrefix("composeResources/").substringAfter("/")
```

Comment:
I find this line too dangerous. It mixes `removePrefix` and `substringAfter` that is hard to predict a final result:
`"composeResources/foo/bar/baz" -> "bar/baz"`
`"/foo/bar/baz" -> "foo/bar/baz"`
`"/composeResources/foo/bar/baz" -> "composeResources/foo/bar/baz"`

Could you please refactor it to use Path API and add a comment here what and why was fixed, please.